docs/tech-notes/life_of_a_query.md

```diff
@@ -186,30 +181,30 @@ that will produce the desired result. There can be a large number of
 functionally equivalent plans for a given query, with very different performance
 characteristics, and the optimizer will try to figure out the most efficient
 plan. Query planning is a complex topic that could easily fill several articles
-of its own, so we won't go into any depth here -- see the
+of its own, so we won't go into any depth here — see the
 [`opt` package documentation](https://github.com/cockroachdb/cockroach/blob/c097a16427f65e9070991f062716d222ea5903fe/pkg/sql/opt/doc.go)
 for further details.
 
 The query plan for a SQL statement can be inspected via `EXPLAIN`. Let's
 have a look at a concrete example:
 
 ```sql
-root@:26257> CREATE TABLE countries (id STRING PRIMARY KEY, name STRING);
-root@:26257> INSERT INTO countries VALUES
+> CREATE TABLE countries (id STRING PRIMARY KEY, name STRING);
```

Comment:
Without the `>` someone might be able to copy-paste these into a local demo shell, which would be helpful.

Reply:
Yeah, I guess it's clear enough what's input and output here.